packages/docusaurus/src/commands/start.ts

```diff
@@ -117,6 +117,12 @@ export async function start(
   );
 
   let config: webpack.Configuration = merge(await createClientConfig(props), {
+    watchOptions: {
+      ignored: [siteDir+'/**/node_modules'],
```

Comment:
```suggestion
      ignored: /node_modules\/(?!@docusaurus)/,
```
We already have this regexp later in the config (static files watch)